### DISCUSSION TITLE
NFTs vs Social Media Platform

### DISCUSSION THREAD

**Discussion by blackstock** [deso-protocol/dips#127] [Medium CIPs]
I think everyone who invested in this project was excited in a decentralized social media platform(s) with built in monetization.

At this stage, I think starting both a social media platform + NFT marketplace is a losing battle. Both spaces are too competitive and have established leaders.

Why not pick a lane? I think focusing on social (decentralized Twitter for now) makes the most sense. If the community wants NFTs, simple, just integrate with the multi-billion dollar NFT marketplace and millions of users on Ethereum. There is already a massive ecosystem with advanced tooling. Why reinvent the wheel?

Bitclout is now going to war against big tech AND the businesses leading the NFT charge in the Ethereum space (OpenSea, SuperRare, Collabland, lending platforms, etc). Why not encourage and give all of the companies building in the Ethereum NFT space a reason to use and promote Bitclout... 

Twitter should be the roadmap for now. Just sprinkle in some monetization. 

Connect your accounts using MetaMask and call it a day. Then the project can leverage every dollar that's going into the amazing NFT ecosystem on Ethereum and focus on social media.

**Comment by tijno** [comment]
What is the change you are recommending ?

This seems more like a statement.

Re NFTs on Bitclout:

NFTs are live on bitclout and I think they innovated in a number of areas over other NFT solutions so I see it as a valuable addition.

I also believe there are further innovations possible to further differentiate bitclout NFT offer.

**Comment by stet** [comment]
"NFT" is a staple feature. You are looking at it the wrong way (competitors). Every platform, crypto and legacy, will support "NFT". So of course BitClout should too.

**Comment by oO** [comment]
I think the core problem is that NFT Collections appear on a feed as a series of individual posts, instead of a single post. Since minting large collections of unique ( single edition ) NFTs is the norm, you can get into situations where an artist you follow, literally destroys your feed with 10s or 100s of freshly minted NFTs.

having a single post saying "So and So just minted 20 new NFTs" would solve the problem nicely.
